spec/services/discovery_engine/sync/delete_spec.rb

```diff
@@ -21,7 +19,7 @@
 
   context "when the delete succeeds" do
     before do
-      delete.call("some_content_id", payload_version: "1")
+      described_class.new("some_content_id", payload_version: "1", client:).call
```

Comment:
Nice! I do have a personal preference for using `described_class` rather than `subject`.